### ISSUE TITLE
Unable to apply different wallpapers on two monitors

### ISSUE THREAD

**Issue by known-alias** [jtsiomb/xlivebg#11]
This was reported by mail a few days ago.

While applying a wallpaper on a dual monitor setup the wallpaper from screen 1 is duplicated in screen 2 even if the wallpaper used is using a resolution meant to be used for two monitors (e.g. 3840 x 1020), everything works as normal except for having the same image in the background.

**Comment by jtsiomb** [comment]
Yes, xlivebg exposes the position and size of each monitor to the live wallpaper plugins, and it's up to the plugins what they do with it. The way this is handled by all plugins currently is to simply show the same thing in every monitor.

It would be nice to have an option available to the plugins, specifying whether the user wants this behavior, or a single large "view" to span all available monitors, so I'm marking this as a feature request. But in the end it will always be up to each plugin whether they heed such an option or not.
